Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "amplify"
-version = "3.10.0"
+version = "3.11.0"
```

Comment:
Since we re-export `amplify_num`, which does a breaking change, we have to do a major version increase here, i.e. go from `3.10.0` to `4.0.0`.

I propose for now just do a `amplify_num` release and leave `amplify` untouched, since we probably will have more breaking changes in `num` part soon. This will requrie direct import of `amplify_num` in AluVM, but we can do that.